src/components/PluginGovernanceStatus.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function PluginGovernanceStatus({plugin}) {
+    if (!plugin || !plugin.labels) {
+        return null;
+    }
+    return plugin.labels.map((id) => {
+        if (id === 'adopt-this-plugin' || id === 'adopt-me') {
```

Comment:
If the plugin has both the `adopt-this-plugin` and `adopt-me` labels, do you want to show it twice?

maybe its better to do `if (plugin.labels.includes('adopt-this-plugin') || plugin.labels.includes('adopt-me')) {` so it only shows up once?

Reply:
I would rather prefer to get rid of adopt-me alltogether

Reply:
yea, like i said, the current code will show the banner twice. so just swap the map out with an if check and you'll be fine.